AUTHORS

```diff
@@ -3,4 +3,5 @@
 
 # Initial version authors:
 ChaiShushan <chaishushan@gmail.com>
+MohamedHelala <mohamed.helala@gmail.com>
```

Comment:
According to the version of this repo, which is a forked version, I think it would be more appropriate to add your name as
```
# Forked version authors:
<your name here>
```

Also, if possible, could you please also help add other contributors into this list? The contributors list can be found at https://github.com/lazywei/go-opencv/graphs/contributors

Thanks.